### ISSUE TITLE
How do you see the file name you opened?

### ISSUE THREAD

**Issue by kdawg1406** [josdejong/jsoneditor#517]
This is probably a dumb question, but after I open the file, I can't find the opened file name in the UI.

Is it available?

Best,

Karl

**Comment by josdejong** [comment]
dumb questions don't exist :) 

you can indeed not see the name of a file after loading it, only when you have saved a file online you will see the file name.

**Comment by kdawg1406** [comment]
Any chance to add it?

**Comment by josdejong** [comment]
Yes that makes sense, I will mark this as a feature request. Not sure about when I have time to implement it though

**Comment by slash-arun** [comment]
Any thoughts on where this is expected to be shown ? Should it be in the same place where the filename is shown currently when saved online ?

**Comment by josdejong** [comment]
I would love to give the web application a big overhaul but haven't found time for it so far.
